genesis-custom-blocks.php

```diff
@@ -7,7 +7,6 @@
  * @license http://opensource.org/licenses/GPL-2.0 GNU General Public License, version 2 (GPL-2.0)
  *
  * Plugin Name: Genesis Custom Blocks
- * Plugin URI: https://studiopress.com
```

Comment:
There was an error in submitting this to wp.org, as the Plugin URI was the same as the Plugin Name